x-pack/plugins/canvas/server/lib/__tests__/create_handlers.js

```diff
@@ -64,14 +64,15 @@ describe('server createHandlers', () => {
     });
 
     it('rejects when authentication check fails', () => {
+      const internalErrorMessage = 'Authentication failed';
```

Comment:
Shouldn't this be a constant somewhere?  It's kind of a brittle test... perhaps an i18n key somewhere?  We should stay ahead of additions while there are so many PRs converting string sites.